src/SldStyleParser.ts

```diff
@@ -439,7 +439,12 @@ class SldStyleParser implements StyleParser {
           break;
         case 'GraphicStroke':
           lineSymbolizer.graphicStroke = this.getPointSymbolizerFromSldSymbolizer(
-            _get(sldSymbolizer, 'Stroke[0].GraphicStroke[0]')
+            sldSymbolizer.Stroke[0].GraphicStroke[0]
+          );
+          break;
+        case 'GraphicFill':
+          lineSymbolizer.graphicFill = this.getPointSymbolizerFromSldSymbolizer(
+            sldSymbolizer.Stroke[0].GraphicFill[0]
```

Comment:
I would prefer usage of `_get()` here (and above). Is there any reason you removed it?

Reply:
Nope, your're right! My bad, haven't taken the context into account.